doc/source/reference/migration_guide_from_1.x_to_2.0.rst

```diff
@@ -0,0 +1,117 @@
+This is a guide for people moving from NetworkX 1.X to NetworkX 2.0
+
+As you know we have made some major changes to the methods in the Multi/Di/Graph classes.
+Methods that used to return lists are removed and the methods that returned iterators
```

Comment:
```Methods that used to return lists are removed and the methods that returned iterators are renamed(removing the _iter suffix).```

It's probably just semantics but G.nodes() isn't actually removed in the sense that it still exists. Also, using "container" instead of "list" includes all the degree() methods as well.

Is it more accurate to say that:
```Methods that used to return containers now return iterators and methods that returned iterators have been removed.```